src/Tribe/PUE/Notices.php

```diff
@@ -197,6 +197,49 @@ public function setup_notices() {
 		}
 	}
 
+	/**
+	 * Select all products with empty license keys
+	 * and format their names
+	 *
+	 * This information will be used to remove products
+	 * with no license keys from $this->notices['invalid_key']
+	 *
+	 * @return array
+	 */
+	public function select_empty_keys() {
+		/** @var $wpdb */
+		global $wpdb;
+
+		$sql = "
+			SELECT option_name
+				FROM {$wpdb->options}
+				WHERE option_name LIKE 'pue_install_key_%'
+					AND option_value=''
+			";
+
+		$empty_keys = $wpdb->get_results( $sql, ARRAY_N );
+
+		$plugin_names = [
```

Comment:
Please use `array()` so we can be compatible with the PHP versions we need to be.

Reply:
:+1: